modules/process-file.js

```diff
@@ -37,14 +44,15 @@ module.exports = ({file, excludeUsers, publish, currentAuthor, config}) => {
         }
 
         return accum;
+
       }, {});
 
       Object.keys(emails).forEach((email) => {
         // { size, ownedLines, author, file_path }
-        checkOwnership({
-          size: lines, 
+        utils.checkOwnership({
```

Comment:
same comment about the named export vs the default. Also utils implies that there are more utils in the module other than checkOwnership.

Reply:
`utils` is kinda just a random name, could be anything else really. The reason is the same as above.